addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -790,16 +790,15 @@ export function getDeepRange(editable, { range, sel, splitText, select, correctT
     // at the last position of the previous node instead.
     const endLeaf = firstLeaf(end);
     const beforeEnd = endLeaf.previousSibling;
+    const previous = previousLeaf(endLeaf, editable, true);
     if (
         correctTripleClick &&
         !endOffset &&
         (start !== end || startOffset !== endOffset) &&
-        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd)))
+        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd))) &&
+        previous && !closestElement(previous, 'table') && closestElement(previous).isContentEditable
     ) {
-        const previous = previousLeaf(endLeaf, editable, true);
-        if (previous && closestElement(previous).isContentEditable) {
-            [end, endOffset] = [previous, nodeSize(previous)];
-        }
+        [end, endOffset] = [previous, nodeSize(previous)];
```

Comment:
Is this still required with the new specs?

Reply:
Yes, it's still required.